packages/core/test/icon/iconTests.tsx

```diff
@@ -14,16 +14,13 @@ import { Classes, Icon, IIconProps, Intent } from "../../src/index";
 
 describe("<Icon>", () => {
     it("iconSize=16 renders standard size", () =>
-        assertIconClass(
-            <Icon iconName="vertical-distribution" iconSize={Icon.SIZE_STANDARD} />,
-            Classes.ICON_STANDARD,
-        ));
+        assertIconSize(<Icon iconName="vertical-distribution" iconSize={Icon.SIZE_STANDARD} />, Icon.SIZE_STANDARD));
 
     it("iconSize=20 renders large size", () =>
-        assertIconClass(<Icon iconName="vertical-distribution" iconSize={Icon.SIZE_LARGE} />, Classes.ICON_LARGE));
+        assertIconSize(<Icon iconName="vertical-distribution" iconSize={Icon.SIZE_LARGE} />, Icon.SIZE_LARGE));
 
-    it("iconSize=inherit renders auto-size", () =>
-        assertIconClass(<Icon iconName="vertical-distribution" iconSize="inherit" />, Classes.ICON));
+    // it("iconSize=inherit renders auto-size", () =>
```

Comment:
?

Reply:
the old `Icon` component supported a special `"inherit"` size which worked nicely with fonts. doesn't work so well with SVG because they're special snowflakes and can inherit non-square sizes (which font glyphs couldn't do - always square)

Reply:
sounds like we should remove this test then